src/Uno.UI/UI/Xaml/Controls/ToolTip/ToolTipService.cs

```diff
@@ -272,6 +274,7 @@ private static void OnPointerPressed(object sender, PointerRoutedEventArgs e)
 				if (e.GetCurrentPoint(owner).Properties.IsLeftButtonPressed)
 				{
 					toolTip.IsOpen = false;
+					toolTip.CurrentHoverId++;
```

Comment:
https://github.com/unoplatform/uno/blob/eee3d255b2ccac1adc311c61d4d59fc2ee1b2df7/src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ToolTipTests/ToolTip_Tests.cs#L74-L76

https://github.com/unoplatform/Uno.UITest/blob/64c6caccd4b2ede96a6ada265c795acb640b3411/src/Uno.UITest.Puppeteer/SeleniumApp.cs#LL497C4-L497C4

Reply:
It feels like it's hiding an issue with the tests on the iOS then. It probably best to validate the change on iOS explicitly.

Reply:
In previous test, after clicking the button, it is expected that tooltip will appear.
But in true behavior, after clicking the button, tooltip shouldn't be appear until pointer entered again (only hover without clicking)

Reply:
Even on iOS.

https://learn.microsoft.com/en-us/dotnet/api/xamarin.uitest.iapp.tapcoordinates?view=xamarin-uitest-sdk#xamarin-uitest-iapp-tapcoordinates(system-single-system-single)


https://github.com/unoplatform/uno/pull/12419/files/390aee5e33c9e3fcf48748079f60c82aa860fbe4#diff-ef1649477ba6ac8e50556565d5ce6fba198a7c4630d0e13a9f6e8d76ec26d4f1L244-L250

Reply:
@jeromelaban ,  I have to resolve this issue in my app. Could you review this PR? I've already modify the test. Thank you.

Reply:
@jeromelaban ,  I have to resolve this issue in my app with in this week. Could you review this PR? I've already modify the test. Thank you.